packages/ckeditor5-widget/src/widget.js

```diff
@@ -131,11 +131,9 @@ export default class Widget extends Plugin {
 				// All of them must be marked as selected, for instance [<widget></widget><widget></widget>]
 				for ( const value of range ) {
 					const node = value.item;
-
-					// Do not mark nested widgets in selected one. See: #57.
+					// Do not mark nested widgets in selected one. See: #4594
```

Comment:
Please revert the original comment. It's confusing right now that it points to another ticket which is unrelated imho for this part of the code.

I believe this file should remain unchanged.

Reply:
@dawidurbanski it seems like the original comment was referring to the issue from some package and it was before merging our repos and issues into monorepo.

If you open #57 you will get some `resolution:invalid` issue, not related to this part of the code, while #4594 is indeed related to showing nested widgets. Also if you go to https://github.com/ckeditor/ckeditor5-widget/issues/57 you will be redirected to #4594.

Reply:
Yeah, just had a chat with Ewelina. Seems like it's all good.